pkgs/fc/agent/src/fc/maintenance/maintenance.py

```diff
@@ -74,9 +73,6 @@ def request_reboot_for_kvm_environment(log) -> Optional[Request]:
 
     qemu_state_dir = Path(QEMU_STATE_DIR)
 
-    def cold_state_file(stem):
-        return Path(KVM_SEED_DIR) / f"qemu-{stem}-booted"
-
```

Comment:
Yeah, I agree with this - there is something weird where moving this off of tmp is not caused immediately after all reboot currently, so this is currently somewhat sloppy. We're going to be better with the new code and I don't need to make the time in between more perfect than it was previously ... no second order fixes here.